### PR TITLE
header_without_burgerMenu_

### PR DESCRIPTION
**Pull request by Anastasiia-Khmelovska**
added header and footer to index, because router with app are not merged yet.

### CODE REVIEW

**Comment by ostUp** [comment]
> added header and footer to index, because router with app are not merged yet.

Already merged, can you check it?

File: fe/package.json [hunk 1]
```diff
@@ -62,5 +64,9 @@
   "lint-staged": {
     "*.tsx": "npm run lint",
     "*.ts": "npm run lint"
-  }
+  },
+  "description": "This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).",
+  "main": ".eslintrc.js",
+  "author": "",
+  "license": "ISC"
 }
```
Comment (VolodymyrChuyko) [inline fe/package.json:72]:
```suggestion
```
I think it would be safer to remove it

File: fe/src/components/Header/Header.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    background-color: white;
+    align-items: center;
+    box-shadow: 0px 1px 0px #E2E6E9;
+
+    @include onDesktop {
+        height: 64px;
+        padding-left: 24px;
+    }
+
+
```
Comment (VolodymyrChuyko) [inline fe/src/components/Header/Header.scss:21]:
```suggestion
```

File: fe/src/styles/reset.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    margin: 0;
+}
+
+ul {
+    padding: 0;
+}
+
+body {
+  margin: 0;
+}
```
Comment (VolodymyrChuyko) [inline fe/src/styles/reset.scss:15]:
```suggestion
h1,
h2,
h3,
p,
ul,
body {
    margin: 0;
}

ul {
    padding: 0;
}
```

**Review by VolodymyrChuyko — CHANGES_REQUESTED** [review]

**Review by VolodymyrChuyko — APPROVED** [review]
Great. Let's try to merge it

**Pull request merged**